src/patients/related-persons/AddRelatedPersonModal.tsx

```diff
@@ -50,9 +55,13 @@ const AddRelatedPersonModal = (props: Props) => {
               placeholder={t('patient.relatedPerson')}
               onChange={onPatientSelect}
               onSearch={async (query: string) => PatientRepository.search(query)}
-              renderMenuItemChildren={(patient: Patient) => (
-                <div>{`${patient.fullName} (${patient.code})`}</div>
-              )}
+              renderMenuItemChildren={(patients: Patient) => {
+                if (patientCode() === patients.code) {
+                  return <div />
+                }
+
+                return <div>{`${patients.fullName} (${patients.code})`}</div>
```

Comment:
Can we add a test proving that the current patient does not come in this list?

Reply:
Hi @jackcmeyer I'm not too familiar with Jest testing, I can try to figure it out, but i'm having some trouble doing so

Reply:
No worries! Feel free to reach out on slack if you need any help